src/Workspaces/Core/Portable/Rename/Renamer.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.CodeAnalysis.Rename
 {
     public static class Renamer
     {
+        internal static bool isRunningInUnitTests;
```

Comment:
Oh, true I guess.  I was thinking this should only be true for the scope of the call.